scripts/check-docs/src/main.rs

```diff
@@ -0,0 +1,120 @@
+extern crate core;
+
+use std::collections::{HashMap, HashSet};
+use std::env;
+use std::path::Path;
+
+fn main() {
+    let mut stack: Vec<&str> = vec![];
+    let mut valid_anchors: HashMap<&str, HashSet<&str>> = HashMap::new();
+
+    let rg_project = std::process::Command::new("grep")
```

Comment:
L19 mentions `"rg command"`, but here `grep` is called. Should be consistent.

Reply:
I see https://github.com/FuelLabs/fuels-rs/pull/532#discussion_r951077312, so you should just change the `expect` message to `grep`.